blocks/identity-block/features/header-account-action/styles.scss

```diff
@@ -1,23 +1,86 @@
 .xpmedia-subs-header {
   align-items: center;
-  color: #fff;
+  color: $ui-btn-white-color;
   font-size: 0.875rem;
 
   &--button {
     align-items: center;
     background: transparent;
     border: 1px solid rgba(255, 255, 255, 0.5);
-    border-radius: 2px;
-    color: #fff;
-    font-size: 0.875rem;
+    border-radius: 4px !important;
+    color: $ui-btn-white-color;
     font-weight: bold;
     height: 2rem;
     padding: 0 0.5rem;
     text-decoration: none;
 
     &:hover {
-      color: #fff;
-      border: 1px solid #fff;
+      color: $ui-btn-white-color;
+      border: 1px solid $ui-btn-white-color;
+    }
+  }
+
+  .account-button {
+    background-color: $ui-btn-white-color;
+    transition: 500ms;
+
+    .account-text {
+      padding: 0 8px;
+    }
+
+    &:hover {
+      filter: brightness(0.85);
+      cursor: default;
+    }
+
+    @media screen and (max-width: map-get($grid-breakpoints, 'md')) {
+      right: 16px;
+      position: absolute;
+      margin-left: 16px;
+
+      .account-text {
+        display: none;
+      }
+
+      .submenu-caret {
+        display: none;
+      }
+    }
+  }
+
+  .account-menu {
+    display: inline;
+    position: absolute;
+    width: 211px;
+    height: 88px;
+    top: 50px;
+    right: 20px;
```

Comment:
is there a way to do this with flex or other positioning? this looks fairly hard-coded to be in a particular place

Reply:
in `._header-nav.scss` has a margin for the logo that make spacing between the logo and right nav element tricky.

Should defer to design on this and figure out a way to have the logo centered, whether we have a right nav element or not.